pkg/utils/utils.go

```diff
@@ -236,3 +242,42 @@ func DeleteDoubleWhiteSpace(slice []string) []string {
 	}
 	return result
 }
+
+// PopulateNamespacesForResources populates namespaces from config file
+func PopulateNamespacesForResources(include, ignore []string) []string {
+	if len(include) == 1 && include[0] == "all" {
+		if (len(ignore) == 1 && ignore[0] != "") || (len(ignore) > 1) {
+			// include : [all]
+			// ignore : [x] or [x,y,z]
+			// Hence include : ClusterNamespaces [...] - [x]
+			include = SetDifference(ClusterNamespaces, ignore)
+			return include
+		}
+	}
+	return include
+}
+
+// listNamespace lists all namespaces present in the cluster
+func listNamespace(KubeClient kubernetes.Interface) []string {
+	namespaceList, err := KubeClient.CoreV1().Namespaces().List(metaV1.ListOptions{})
+	if err != nil {
+		log.Logger.Error(err.Error())
```

Comment:
We should not just make error here. Can result in unpredictable behavior of BotKube. 
Probably return error and caller should handle it.

Reply:
resolved it